BrowserKit/Sources/ToolbarKit/AddressToolbar/AddressToolbarState.swift

```diff
@@ -7,27 +7,27 @@ import Foundation
 /// Defines the state for the address toolbar.
 public struct AddressToolbarConfiguration {
     /// View config for the `Location View`  in the address toolbar
-    let locationViewConfiguration: LocationViewConfiguration
+    public let locationViewConfiguration: LocationViewConfiguration
```

Comment:
I don't think these should be public. All the config should be in the toolbar state as well.